pepper-apis/configure.rb

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/ruby -w
 # Docker Image used to call out to generate config files
-$dsde_toolbox_image_name = "broadinstitute/study-server-build:java-2022-06-01A"
+
+$dsde_toolbox_image_name = "broadinstitute/study-server-build:java-2023-03-28C"
```

Comment:
We may want to get even more recent versions of deps in dockerfiles.  The readme describes how to label and push these builds so that whenever we're running things on this branch, we can keep the images in sync without cluttering mainline.